run.sh

```diff
@@ -1,31 +1,38 @@
 #!/bin/sh
 
+#add environmental variable of .env to the bash script
+. ./.env
+
+#Define absolute path for the host and the serv from the relative common path
+HOST_LOG_FILE=$PWD${LOG_FILE}
```

Comment:
At least you should do:
```diff
-HOST_LOG_FILE=$PWD${LOG_FILE}
+HOST_LOG_FILE=${PWD}${LOG_FILE}
```

but [`realpath`](https://www.man7.org/linux/man-pages/man1/realpath.1.html) is better!
```suggestion
HOST_LOG_FILE=$(realpath -P ${LOG_FILE})
```

examples:
```
[ctmbl:~] $ realpath -P Documents/myfile
/home/ctmbl/Documents/myfile
```

Reply:
Done in e211d41

Reply:
:rocket: